serialcheck.c

```diff
@@ -417,6 +427,7 @@ static int stress_test_uart(struct g_opt *opts, int fd, unsigned char *data,
 int main(int argc, char *argv[])
 {
 	struct g_opt opts;
+	struct sigaction sigIntHandler;
```

Comment:
Hi, can you get rid of camelCase here? Rest looks okay and will merge. Apologies for missing this so far.

Reply:
Updated